lwk/lwkwallet.go

```diff
@@ -13,9 +14,14 @@ import (
 	"github.com/vulpemventures/go-elements/psetv2"
 )
 
+// Satoshi represents a satoshi value.
+type Satoshi = uint64
+
+// SatPerKVByte represents a fee rate in sat/kb.
+type SatPerKVByte = uint64
+
 const (
-	kiloByte          = 1000
-	minimumSatPerByte = 0.1
+	minimumSatPerByte = 200
```

Comment:
What is the rational behind setting the min sat per byte to 200?

Reply:
This was my mistake and has been corrected to 0.1
https://github.com/ElementsProject/peerswap/pull/288/files#diff-023bf190da1cb5d2f1e97d29c0d4c43624fabb6897db4573293921a3c3dab2cbR28